HW_1/task1.py

```diff
@@ -0,0 +1,70 @@
+print("Task 2")
+# Task 2: Find the sum of the digits of a three-digit number.
+# *Example:*
+# 123 -> 6 (1 + 2 + 3)
+# 100 -> 1 (1 + 0 + 0) |
+three_digit_number = int(input("Enter a three-digit number :"))
+if (three_digit_number > 999) or (three_digit_number < 100):
+    print("Sorry, you enter NOT a three-digit number, start again")
+else:
+    first = int(three_digit_number / 100)
+    second = int((three_digit_number % 100) / 10)
```

Comment:
Функцию int использовать не обязательно. Тут отлично работает оператор целочисленного отделения.
```suggestion
    first = three_digit_number // 100
    second = three_digit_number % 100 // 10
```